lms/util/h.py

```diff
@@ -0,0 +1,55 @@
+# -*- coding: utf-8 -*-
+
+"""Utility functions for working with the h API."""
+
+from __future__ import unicode_literals
+
+import random
+import string
+
+
+ALLOWED_CHARS = string.ascii_letters + string.digits + "._"
+"""The characters that are allowed in h usernames."""
```

Comment:
This is a standard Python "attribute docstring"

Reply:
Cool. :)